routes/people-routes.js

```diff
@@ -0,0 +1,77 @@
+// *********************************************************************************
+// api-routes.js - this file offers a set of routes for displaying and saving data to the db
+// *********************************************************************************
+
+// Dependencies
+// =============================================================
+
+// Requiring our models
+var db = require("../models");
+
+// Routes
+// =============================================================
+module.exports = function(app) {
+
+  // GET route for getting all of the people
+  app.get("/api/people", function(req, res) {
+    var query = {};
+    if (req.query.location_id) {
+      query.LocationId = req.query.location_id;
+    }
+    // Here we add an "include" property to our options in our findAll query
+    // We set the value to an array of the models we want to include in a left outer join
+    // In this case, just db.Location
+    db.Post.findAll({
```

Comment:
I know this was probably just remnants from the example in class repo, but db.Post should probably be db.People